### PR TITLE
BAEL-7676: Get the Response Body in Spring Boot Filter

### PR DESCRIPTION
**Pull request by harry9656**

### CODE REVIEW

File: spring-boot-modules/spring-boot-basic-customization-3/src/main/java/com/baeldung/responsebody/MD5Filter.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+import jakarta.servlet.ServletRequest;
+import jakarta.servlet.ServletResponse;
+import jakarta.servlet.http.HttpServletResponse;
+import jakarta.xml.bind.DatatypeConverter;
+
+// MD5Filter.java
+@Component
+public class MD5Filter implements Filter {
+
+    Logger log = Logger.getLogger(MD5Filter.class.getName());
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-basic-customization-3/src/main/java/com/baeldung/responsebody/MD5Filter.java:23]: It seems log is never used.

Comment (harry9656) [reply spring-boot-modules/spring-boot-basic-customization-3/src/main/java/com/baeldung/responsebody/MD5Filter.java]: I removed it.

File: spring-boot-modules/spring-boot-basic-customization-3/src/test/java/com/baeldung/responsebody/ResponseBodyUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+@SpringBootTest(classes = ResponseBodyApplication.class)
+@AutoConfigureMockMvc
+class ResponseBodyUnitTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Test
+    void testMD5FilterForGetRequest() throws Exception {
```
Comment (JavierPeris) [inline spring-boot-modules/spring-boot-basic-customization-3/src/test/java/com/baeldung/responsebody/ResponseBodyUnitTest.java:29]: Please name the test following the convention: givenXX_whenXXX_thenXXX

Comment (harry9656) [reply spring-boot-modules/spring-boot-basic-customization-3/src/test/java/com/baeldung/responsebody/ResponseBodyUnitTest.java]: Renamed it.

**Pull request merged**